services/chrome-web-store/chrome-web-store-size.service.js

```diff
@@ -22,8 +22,8 @@ export default class ChromeWebStoreSize extends BaseChromeWebStoreService {
     color: 'blue',
   }
 
-  transform(sizeStr) {
-    const match = sizeStr.match(/^(\d+)([a-zA-Z]+)$/)
+  static transform(sizeStr) {
+    const match = sizeStr.match(/^([\d.]+)([a-zA-Z]+)$/)
```

Comment:
```suggestion
    const match = sizeStr.match(/^(\d+(?:\.\d+)?)([a-zA-Z]+)$/)
```
This will avoid multiple dots/numbers in string and is more accurate.
number & optional .number
while not effecting the captured groups.